github/src/main/kotlin/gropius/sync/github/TimelineItemHandler.kt

```diff
@@ -31,7 +31,7 @@ class TimelineItemHandler(
      * @return the neo4j-id for the created item (if created) and the last DateTime concerning this item
      */
     private suspend fun handleIssueComment(
-        issue: IssueInfo, event: IssueCommentTimelineItemData
+        imsProjectConfig: IMSProjectConfig, issue: IssueInfo, event: IssueCommentTimelineItemData
```

Comment:
these new params seem to not have made it into the docs

Reply:
97e7d2bad00bc5df055901a6122ebc81ba380105